commands/build.go

```diff
@@ -737,8 +737,30 @@ func checkWarnedFlags(f *pflag.Flag) {
 	}
 }
 
-func writeMetadataFile(filename string, dt interface{}) error {
-	b, err := json.MarshalIndent(dt, "", "  ")
+func writeMetadataFile(filename string, response map[string]interface{}) error {
+	out := make(map[string]interface{})
+	for k, v := range response {
+		var dt []byte
+		j, ok := v.(string)
+		if ok {
+			d, err := base64.StdEncoding.DecodeString(string(j))
```

Comment:
There is already a decoder in `decodeExporterResponse` in the same code path, so it shouldn't be duplicated.